src/node_api/indexer/routes.rs

```diff
@@ -20,7 +20,8 @@ impl Client {
     pub async fn output_ids(&self, query_parameters: Vec<QueryParameter>) -> Result<Vec<OutputId>> {
         let route = "api/plugins/indexer/v1/outputs/basic";
 
-        self.get_output_ids_with_pagination(route, query_parameters).await
+        self.get_output_ids_with_pagination(route, query_parameters, false, false)
```

Comment:
I would say for all indexer routes quorum should be true, since we only get unspent outputs and not spent outputs anymore they should return always the same thing